### DISCUSSION TITLE
table 分页栏的，每页显示条数limit，怎么获取选择后的数值呢？

### DISCUSSION THREAD

**Discussion by wangwenjie7** [layui/layui#1877] [问答]
![GqgUNfN6hX](https://github.com/layui/layui/assets/29366022/4eb7c853-022f-4560-9180-13b072680a1f)
或者说，我修改表的某条数据，保存后，怎样仍然 固定在 当前页面 curr 和 当前选择显示的每页条数 limit

**Comment by Sight-wcg** [comment]
> 每页显示条数limit，怎么获取选择后的数值呢

参考下面的代码，根据你的业务选择合适的方式获取分页信息
```
var tbInstance = table.render({
  done: function(){
    var tablebOptions = this;
    console.log(tablebOptions.page)
  }
})

var tbConfig = tbInstance.config
console.log(tbConfig.page)
```
